client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -331,7 +331,7 @@ export default function WPCheckout( {
 	const { transactionStatus } = useTransactionStatus();
 	const paymentMethod = usePaymentMethod();
 	const shouldCollapseLastStep = useShouldCollapseLastStep();
-	const showToSFoldableCard = useToSFoldableCard();
+	const showToSFoldableCard = useToSFoldableCard() === 'treatment';
```

Comment:
This is ok as long as you can ensure that this will never return loading at this point.